{{ cookiecutter.package_name }}/docs/index.rst

```diff
@@ -1,14 +1,17 @@
-Documentation
-=============
+{{ '*' * "%s Documentation"|format(cookiecutter.package_name)|length() }}
+{{ cookiecutter.package_name }} Documentation
+{{ '*' * "%s Documentation"|format(cookiecutter.package_name)|length() }}
 
 This is the documentation for {{ cookiecutter.package_name }}.
 
 .. toctree::
   :maxdepth: 2
 
-  {{ cookiecutter.module_name }}/index.rst
+  subpage.rst
+
+.. note:: The layout of this directory is simply a suggestion.
```

Comment:
I would just remove this note - in fact the presence of Makefile, index.rst and so on is not really a suggestion - the way you have it now is pretty standard.

Reply:
OK, removed.